packages/wallet-standard/src/wallet.ts

```diff
@@ -106,14 +103,16 @@ export class GlowWallet implements Wallet {
     return this.#account ? [this.#account] : [];
   }
 
-  constructor() {
+  constructor(glow: GlowAdapter) {
```

Comment:
We pass the reference in instead of using `window.glow`. This means that even if another extension or library writes to the global, Glow's standard wallet will work.